README.md

```diff
@@ -171,16 +173,33 @@ mstorage.list(){result, error in
         // do something
     }
 }
-// You can also use a Dictionary object for listing options as follows.
-// The available options are "limit", "after" and "before".
+```
+
+* With options
+
+You can also use a `Dictionary` object for listing options as follows.
+The available options are `limit`, `after` and `before`.
+
+```swift
 mstorage.list(["limit": "25", "after": "<media_id>"]){result, error in
     // do something
 }
 ```
 
-### Delete
+* Search
+
+Search media ids by user metadata
+
+```swift
+mstorage.list(["limit": "25", "after": "<media_id>",
```

Comment:
As we discussed on Slack, we don't need to fix this. Logs are following: https://mandarinduck.slack.com/archives/neco/p1468215349000076